qingstor/sdk/utils/file_chunk.py

```diff
@@ -82,3 +87,49 @@ def __next__(self):
 
     def __iter__(self):
         return self
+
+
+class EncryptionFileChunk(FileChunk):
+    def __init__(self,fd,key):
```

Comment:
We need a `encryption_algorithm` here to support more encrypt algorithm.

And we need a doc to inform users that wo only support AES256 for now.